fe/src/pages/About/About.tsx

```diff
@@ -1,9 +1,11 @@
+import { Box } from '@chakra-ui/react';
+
 function About(): JSX.Element {
   // @TODO - Add some actual content here.
   return (
-    <div>
+    <Box h="100vh">
```

Comment:
Temporary visual change to make footer position lower for "empty" pages.

Reply:
https://jsfiddle.net/wnLT7/6/

Reply:
With Kaspars approach you also have to add padding to the wrapper so the footer wouldn't cover it up when you have longer content.

Reply:
Went with @Kasparsu approach, check above for updated visual picture.